src/SFA.DAS.Tools.Support.Infrastructure/SFA.DAS.Tools.Support.Infrastructure.csproj

```diff
@@ -1,12 +1,15 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.1</TargetFramework>
+    <TargetFramework>netstandard2.0</TargetFramework>
   </PropertyGroup>
 
   <ItemGroup>
     <PackageReference Include="AutoMapper" Version="9.0.0" />
-    <PackageReference Include="SFA.DAS.CommitmentsV2.Api.Client" Version="5.4.0" />
+    <PackageReference Include="Microsoft.Extensions.Logging" Version="3.1.12" />
+    <PackageReference Include="SFA.DAS.Account.Api.Client" Version="1.6.2978" />
+     <PackageReference Include="SFA.DAS.EmployerUsers.Api.Client" Version="1.1.309-prerelease-80" />
```

Comment:
It would be good to replace prerelease package but employer users are still building from the branch

Reply:
Will update it when the new version is out of pre-release status.